llvm/include/llvm/TargetParser/X86TargetParser.def

```diff
@@ -261,6 +261,7 @@ X86_FEATURE_COMPAT(AVX10_1,         "avx10.1-256",           36)
 X86_FEATURE_COMPAT(AVX10_1_512,     "avx10.1-512",           37)
 X86_FEATURE_COMPAT(AVX10_2,         "avx10.2-256",            0)
 X86_FEATURE_COMPAT(AVX10_2_512,     "avx10.2-512",            0)
+X86_FEATURE_COMPAT(MOVRS,           "movrs",                  0)
```

Comment:
Why putting it before ZU and using `X86_FEATURE_COMPAT`?

Reply:
I suppose gcc would put it following AVX10_2_512 but we'd better wait and sync with them.

Reply:
[3a4f4b7](https://github.com/llvm/llvm-project/pull/113274/commits/3a4f4b76754ddec77bb013f0c1896e2495f3b1cb) changed it into X86_FEATURE and added a fixme.
gcc patch: https://gcc.gnu.org/pipermail/gcc-patches/2024-October/666043.html